NuGet.Config

```diff
@@ -10,6 +10,7 @@
       Used for azure-sdk-tools repo until issue https://github.com/Azure/azure-sdk-tools/issues/1329 is addressed
     -->
     <add key="azure-sdk-tools" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json" />
+    <add key="air-msrai-decisionservice-Consumption" value="https://msazure.pkgs.visualstudio.com/one/_packaging/air-msrai-decisionservice-Consumption/nuget/v3/index.json" />
```

Comment:
[revert] We are not to have multiple NuGet sources in a build. Please make RLNet a downstream consumption of azure-sdk-tools.

Reply:
We are not there yet. We need to first build the Nuget package and provide the package file to dotNet team. Then they can put it under their dev feed: https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-net.  Then we will update this line to point to the dev feed. Currently we still don't have a good Nuget package. This is still a temporary location just to make sure we can continue our implementation.